### DISCUSSION TITLE
Need "OnTap Event" for Map Interactive Marker

### DISCUSSION THREAD

**Discussion by Michel7GitHub** [flet-dev/flet#3726] [Q&A] [answered]
Here is an example of what I would like to present in my App : https://developers.google.com/maps/documentation/javascript/examples/advanced-markers-html
To succeed, it seems to be missing "OnTap Event" for Map Marker.

**Comment by ndonkoHenri** [comment, marked as answer]
`Marker.content` can literally be any Flet control. 

Rough suggestion: wrap the actual content you have (ex: an icon) into a control which can listen to click/tap events(`Container`, `GestureDetector`, ...) and listen to taps/clicks from there.
